packages/assets/assets/full-screen.svg

```diff
@@ -1,4 +1,4 @@
-<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<svg width="100%" height="100%" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Let's change the `stroke` props here to `#0D0D0D`. The image is white when previewed from the IDE/GitHub.